app/models/progress_message.rb

```diff
@@ -13,6 +13,8 @@
 class ProgressMessage < ApplicationRecord
   acts_as_tenant(:tenant)
 
+  validates :owner, :presence => true
```

Comment:
I am not seeing the need for ownership on this table.  Ownership could be derived from the relationship to the order, plus messages here could be system generated.